Packs/DomainToolsIrisDetect/Integrations/DomainToolsIrisDetect/DomainToolsIrisDetect.yml

```diff
@@ -0,0 +1,1349 @@
+  defaultvalue: Create Incidents and Import Indicators
+  display: Enabled on New Domains
+  hidden: false
+  name: new_domains
+  options:
+  - Import Indicators Only
+  - Create Incidents and Import Indicators
+  required: false
+  type: 15
+- additionalinfo: Monitors the Iris Detect endpoint for recent changes to domains added to the watchlist. This is useful for monitoring changes to infrastructure after a domain has been triaged from the "new" endpoint or within the Iris Detect UI. If selected, each pull will create a new incident every time the enrichment is run, with the new domains attached as indicators to the incident. Whois and DNS information is preserved in comments.
+  defaultvalue: Create Incidents and Import Indicators
+  display: Enabled on Changed Domains
+  hidden: false
+  name: changed_domains
+  options:
+  - Import Indicators Only
+  - Create Incidents and Import Indicators
+  required: false
+  type: 15
+- additionalinfo: Monitors the Iris Detect endpoint for additions to domains added to the blocklist. Additions can be made via this app or the Iris Detect UI. This is useful for teams wishing to route triage domains to firewall software for blocking. If selected, each pull will create a new incident every time the enrichment is run, with the new domains attached as indicators to the incident. Whois and DNS information is preserved in comments.
+  defaultvalue: Create Incidents and Import Indicators
+  display: Enabled on Blocked Domains
+  hidden: false
+  name: blocked_domains
+  options:
+  - Import Indicators Only
+  - Create Incidents and Import Indicators
+  required: false
+  type: 15
+- additionalinfo: Optionally specify a risk score range to triage higher risk indicators to different routing. A higher number indicates higher confidence a domain is likely to be used for malicious purposes.
+  display: Risk score Ranges
+  hidden: false
+  name: risk_score_ranges
+  options:
+  - 0-0
+  - 1-39
+  - 40-69
+  - 70-99
+  - 100-100
+  required: false
+  type: 15
+- additionalinfo: Includes DNS and whois data in the response.
+  defaultvalue: 'true'
+  display: Include Domain Data
+  hidden: false
+  name: include_domain_data
+  required: false
+  type: 8
+- additionalinfo: For the first time the enrichment is run, specify how far back should it pull indicators. First Fetch timestamp, Default is 3 days. The maximum time range is 30 days.
+  defaultvalue: 3 days
+  display: First fetch timestamp (example 7 days)
+  name: first_fetch
+  required: false
+  type: 0
+- additionalinfo: This is a required field by XSOAR and should be set to 3, one for each possible feed type new, changed, blocked.
+  defaultvalue: '3'
```

Comment:
Why the default value is 3? 
The default value of the max_fetch parameter should represent how many incident (or domains in your case) to fetch each fetch-incidents.

Reply:
Since we are saving only 3 incidents we have this value, but we are hiding it, since it does not make any sense to provide as user input.